### ISSUE TITLE
New version broke item select from hotbar

### ISSUE THREAD

**Issue by WolfgamingWG** [Strobel273/InventiveInventory#31]
**Mod Version & Minecraft Version**
0.4.3 on 1.21 Fabric

**Describe the bug**
When holding the button binded to advanced operation while holding a item such as a tool, it does not show the other items (tools) anymore. Same with things like 

**To Reproduce**
1. Hold item
2. Press keybind for advanced operation
3. See error

**Expected behavior**
When holding advanced operation button it should show related items

**Screenshots**
Basically, this is missing: https://cdn.discordapp.com/attachments/1219399383003430943/1252356629773680721/20240617_221612.jpg?ex=6671eb80&is=66709a00&hm=87a491ccfac424032b168bc336a1429656d1b71c3da1822bfee6c4fb41322873&

**Comment by Strobel273** [comment]
Hey WolfgamingWG,
thanks for your report.

Unfortunately, we don't think your problem is on our side. A GUI like the one in your screenshot is not familiar to us and the feature you described doesn't suit any of us. Maybe you can give us more detailed information about your problem.

Or maybe it's a compatibility issue with other mods you use. So you could give us a list of the mods you use and we'll check if we can reproduce the problem.

We hope we were able to help you.

Your Inventive Inventory Team

**Comment by WolfgamingWG** [comment]
Hello Strobel

I have now found out that i was way to tired last night and posted this github issue without double checking it was the right mod. my sincerest apologies for my mistake, ill close this issue.
